pkg/controller/certificates/rootcacertpublisher/publisher.go

```diff
@@ -205,13 +211,18 @@ func (c *Publisher) syncNamespace(ns string) (err error) {
 		"ca.crt": string(c.rootCA),
 	}
 
-	if reflect.DeepEqual(cm.Data, data) {
+	// ensure the data and the one annotation describing usage of this configmap match.
+	if reflect.DeepEqual(cm.Data, data) && cm.Annotations[DescriptionAnnotation] == Description {
```

Comment:
if the description text above ever changes, this will duel... should we only care if the description annotation is completely missing?

Reply:
> if the description text above ever changes, this will duel... should we only care if the description annotation is completely missing?

Sure.